src/patternfly/components/Progress/progress__measure.hbs

```diff
@@ -1,4 +1,8 @@
 {{!-- This provides the measure with a % --}}
 {{#if progress__value}}
-  <span class="pf-c-progress__measure">{{progress__value}}%</span>
+  {{~#if progress--dynamic}}
+    <span class="pf-c-progress__measure">{{progress__valuetext}}</span>
+    {{else}}
```

Comment:
Else should be moved back two spaces to align with the if start/end blocks and add visual continuity to the logic functions.

Reply:
Spacing adjusted.